app/services/ExactTargetService.scala

```diff
@@ -50,8 +50,6 @@ trait ExactTargetService extends LazyLogging {
           logger.error(errorMsg)
           throw new ExactTargetException(errorMsg)
```

Comment:
Should you not remove the thrown exception if you've removed the `recover` block? Otherwise you'll change the behaviour - this error will bubble up and possibly cause a 500 error